manifests/install.pp

```diff
@@ -3,7 +3,7 @@
   $ensure  = 'latest',
   ) {
 
-  package { 'scout':
+  package { '^scout':
```

Comment:
I agree. Might be solved in 3.5.0-rc1.
https://github.com/puppetlabs/puppet/commit/77a7b96f585cff78851f7d335d75e0e5b7eedeb5

anything before that would break (if that solves it).